src/blackmisc/simulation/fscommon/fsdirectories.h

```diff
@@ -110,18 +126,37 @@ namespace BlackMisc::Simulation::FsCommon
         //! Get all the SimObjects paths from all config files
         static QSet<QString> allFsxSimObjectPaths();
 
+        // TODO TZ
+        //! Get all the SimObjects paths from all config files
+        static QSet<QString> allMsfsSimObjectPaths();
+
         //! Find the config files (fsx.cfg)
         // C:/Users/Joe Doe/AppData/Roaming/Microsoft/FSX/fsx.cfg
         static QStringList findFsxConfigFiles();
 
+        // TODO TZ
```

Comment:
Is this TODO still outstanding?

Reply:
I think this function doesn't do what it's supposed to do. That's why I want to watch it

Reply:
I think this function doesn't do what it's supposed to do. That's why I want to watch it